css/main.css

```diff
@@ -262,3 +262,447 @@ textarea {
+.features h4 {
+  color: black;
+  font-weight: normal;
+  text-transform: uppercase;
+  padding: 20px 5px 5px;
+  font-family: "Roboto Light", sans-serif;
+}
+
+div.feature {
+  z-index: 999;
+  height: 170px;
+  background-color: white;
+  color: darkgrey;
+  float: left;
+  font-size: 14px;
+  text-align: center;
+  margin: -40px 20px;
+  overflow: hidden;
+  box-shadow: 0px 0px 15px grey;
+  padding: 5px -10px;
+}
+
+.clearfix::after {
+  content: "";
+  display: block;
+}
+div.feature > img {
+  display: flex;
+}
+
+button.arrow {
+  position: absolute;
+  right: 16%;
+  bottom: 22%;
+  border-radius: 45px;
+  height: 35px;
+  width: 35px;
+}
+
+button.arrow > p {
+  line-height: 17px;
+  margin: 0 3px 0 0 ;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+/*Main*/
+
+main {
+  padding: 100px 0;
+}
+
+main > section > h2 {
+  color: black;
+  text-transform: none;
+  text-align: center;
+  font-size: 20px;
+  font-family: sans-serif;
+  text-decoration-line: underline;
+  text-decoration-color: #4574e9;
+  margin: 10px 0;
+}
+
+main > section > p {
+  background: white;
+  color: black;
+  text-align: center;
+  font-family: sans-serif;
+  font-size: 14px;
+}
+
+/*Main - section about*/
+
+section.about > div {
+  position: absolute;
+}
+
+
+div > img {
+  margin-top: 2%;
+  width: 40%;
+  position: relative;
+  left: 30%;
+
+}
+section.about > div > button, button.see-more {
+  font-family: sans-serif;
+  border: 2px solid black;
+  border-radius: 45px;
+  background-color: black;
+  color: white;
+  font-family: sans-serif;
+  text-transform: none;
+  width: 100px;
+  font-size: 10px;
+
+}
+
+section.about > div > h4 {
+  font-family: "Roboto Light", sans-serif;
+  color: #4f4e4e;
+  font-weight: normal;
+  position: absolute;
+  top: 28%;
+  left: 38%;
+}
+
+section.about > div > p {
+  position: absolute;
+  top: 36%;
+  left: 36%;
+  font-family: "Roboto Light", sans-serif;
+  font-size: 70%;
+  color: #383838;
+  width: 26%;
+  text-align: justify;
+  margin: 0 10px;
+}
+section.about > div > button {
+  position: absolute;
+  bottom: 14%;
+  left: 46%;
+}
+
+/*Main-section What we do*/
+section.we-do {
+  border-bottom: 1px solid rgb(189, 188, 188);
+}
+
+section.we-do > h2 {
+  margin-top: 400px;
+}
+
+.we-do > p {
+  font-weight: bold;
+}
+
+div.images {
+  margin: 50px 142px 0;
+  width: 55%;
+}
+
+button.see-more {
+  margin: 50px 580px;
+}
+
+
+/*Section testimonial*/
+
+div.jckolo, div.rokoyo {
+  float: left;
+  justify-content: space-between;
+  width: 28%;
+  height: 200px;
+  background-color: #17122b;
+  padding: 15px 12px;
+  margin-top: 80px;
+  margin-left: 20px;
+}
+
+
+div.jckolo {
+  margin-left: 21.3%;
+}
+
+div.jckolo > img, div.rokoyo > img {
+  position: relative;
+  width: 20%;
+  left: 0%;
+}
+div.jckolo > h6, p.subtitle, p.description, div.rokoyo > h6, p.subtitle, p.description {
+  color: rgba(245, 245, 245, 0.94);
+  font-family: Calibri;
+  font-size: 15px;
+  position: relative;
+  top: -35%;
+  left: 25%;
```

Comment:
W wielu miejscach ręcznie ustawiasz procentami pozycję elementu. Powinnaś tego unikać i używać tak naprawdę, gdy masz jakiś element ustawiony absolutnie albo fixed. Jakbyś musiała zrobić do tego RWD i ustawiać te pozycje ręcznie to po tygodniu szarpania wyrzuciłabyś kompa przez okno

Reply:
co to jest rwd?